styles/globals.css

```diff
@@ -74,9 +71,210 @@ h3 {
+}
+
+.reviewSwiper .slideHandle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.reviewSwiper::after,
+.reviewSwiper::before {
+  content: '';
+  background: #fff;
+  z-index: 2;
+  width: 300px;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  filter: blur(10px);
+}
+
+.reviewSwiper::before {
+  background: linear-gradient(to right, #fff, rgba(255, 255, 255, 0.95));
+  left: 0;
+}
+
+.reviewSwiper::after {
+  right: 0;
+  background: linear-gradient(to left, #fff, rgba(255, 255, 255, 0.95));
+}
+
+@media screen and (max-width: 1050px) {
+
+  .reviewSwiper::after,
+  .reviewSwiper::before {
+    display: none;
+  }
+}
+
+.reviewSwiper .swiper-slide .slideContainer {
+  opacity: 0.6;
+  transition: all 0.8s ease-in-out;
+}
+
+@media screen and (max-width: 1024px) {
+  .reviewSwiper .swiper-slide .slideContainer {
+    opacity: 1;
+  }
+}
+
+.reviewSwiper .swiper-slide-active .slideContainer {
+  opacity: 1;
+  transform: scale(1.2);
+}
+
+.reviewSwiper .swiper-pagination-bullet {
+  width: 15px;
+  opacity: 1;
+  height: 15px;
+  border: 2px solid #c7cbd6;
+  background: transparent;
+}
+
+.reviewSwiper .swiper-pagination-bullet-active {
+  background: #c7cbd6;
+}
+
+/* Exclusive Stockist */
+.exclusiveContainer::before {
+  content: "";
+  display: block;
+  opacity: 0;
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #c7cbd640;
+  transition: all .4s;
+  z-index: 2;
+}
+
+.exclusiveContainer:hover:before {
+  opacity: 1;
+}
+
+.exclusiveLinkIcon {
+  display: none !important;
+}
+
+.exclusiveContainer:hover .exclusiveLinkIcon {
+  display: block !important;
+}
+
+/* About Us */
+
+.aboutus img {
+  width: 33.33%;
+}
+
+.aboutus h1 {
+  margin: 40px 0 20px;
+  font-weight: 100;
+  text-align: left;
+  font-size: 2rem;
+  font-weight: 100;
+  color: #3e3d3c;
+}
+
+.aboutus .beauti {
+  white-space: nowrap;
+}
+
+
+.aboutus .beauti div {
+  font-weight: 100;
+  font-size: 1.5rem;
+  font-size: 29px;
+  display: flex;
+  place-items: baseline;
+  gap: 0.5rem;
+  margin-bottom: 40px;
+}
+
+.aboutus .beauti div h2 {
+  font-weight: 400;
+  color: #3e3d3c;
+}
+
+@media screen and (max-width: 768px) {
+  .aboutus .beauti div {
+    flex-direction: column;
+  }
+  .aboutus .beauti div {
+    font-size: 1rem;
+  }
+  .aboutus .beauti div h2 {
+    padding-left: 1rem;
+  }
+}
+
+.aboutus h3 {
+  text-align: start;
+  font-size: 17px;
+  font-weight: 100;
+  line-height: 32px;
+  margin-bottom: 40px;
+  color: #3e3d3c;
+}
+
+.aboutus a {
+  text-decoration: underline;
+  color: #3e3d3c;
+}
+
+.aboutus p {
+  margin-bottom: 40px;
+  font-size: 17px !important;
+  font-weight: 100;
+  line-height: 32px;
+  color: #3e3d3c;
+}
+
+.aboutus h4 {
+  font-weight: 300;
+  font-size: 29px;
+  margin: 20px 0;
+  text-align: left;
+  color: #3e3d3c;
+}
+
+.aboutus h5 {
+  font-weight: 100;
+  font-size: 29px;
+  margin: 20px 0;
+  text-align: left;
+  color: #3e3d3c;
+}
```

Comment:
Could this be moved in the related components as a style component wrapper?

Reply:
No, this can't be moved to the component files. It needs to be in a .css file because of multiple reasons. I can turn the queries into one line of Tailwind, it'll help in reducing lines of code